lib/commands/bootstrap.js

```diff
@@ -25,19 +24,24 @@ module.exports = {
 		return new Promise( ( resolve, reject ) => {
 			const destinationPath = path.join( data.options.packages, data.repository.directory );
 
+			let promise;
+
 			// Package is already cloned.
 			if ( fs.existsSync( destinationPath ) ) {
-				log.info( `Package "${ data.packageName }" is already cloned. Skipping.` );
+				log.info( `Package "${ data.packageName }" is already cloned.` );
 
-				return resolve( { logs: log.all() } );
-			}
+				promise = Promise.resolve();
+			} else {
+				const command = [
+					`git clone --progress ${ data.repository.url } ${ destinationPath }`,
+					`cd ${ destinationPath }`,
+					`git checkout --quiet ${ data.repository.branch }`
+				].join( ' && ' );
 
-			const command =
-				`git clone --progress ${ data.repository.url } ${ destinationPath } && ` +
-				`cd ${ destinationPath } && ` +
-				`git checkout --quiet ${ data.repository.branch }`;
+				promise = exec( command );
+			}
 
-			exec( command )
+			promise
```

Comment:
I see now where resolve and reject parts are. But it looks a little bit strange for me to return a promise which has another promise inside and which is even not returned ;) Maybe it could be written step by step with `.then()` method and get 27-42 lines before the return statement?